core/src/main/java/org/apache/calcite/sql/type/SqlTypeUtil.java

```diff
@@ -1510,6 +1563,31 @@ public static boolean isMap(RelDataType type) {
     return type.getSqlTypeName() == SqlTypeName.MAP;
   }
 
+  /**
+   * @return true if type is MULTISET
+   */
+  public static boolean isMultiset(RelDataType type) {
```

Comment:
I think this function is not used as you are switching to `isCollection`

Reply:
Indeed, I did not use this method, but basically, all RelDataType have a similar utility method, so I added this for the future use. If you think this is unnecessary, I can remove this.

Reply:
For me keeping this is ok.

Reply:
![image](https://user-images.githubusercontent.com/15643702/84861572-ed8fbf00-b0a3-11ea-975a-4cf0ee1a1067.png)